vendor/github.com/docker/distribution/testutil/handler.go

```diff
@@ -0,0 +1,148 @@
+package testutil
```

Comment:
is this actually a carry or just committing an existing package we now depend on?